src/views/Transactions/TransactionHistory.scss

```diff
@@ -13,6 +13,11 @@
 }
 
 .transaction-history {
+  .body-pane {
```

Comment:
I already made a change in `roller` to fix the padding issue (I used a negative margin), sorry, should have tagged you. I think you can remove the changes on this file.